### DISCUSSION TITLE
How to handle versioning?

### DISCUSSION THREAD

**Discussion by erictuvesson** [noodlapp/noodl-docs#24] [General]
In the previous version and the current we have prefixed the hosting URL by the version.

The older versions of the docs were not built on top of Docusaurus so hard to convert all the older versions, and not sure how much work it would be to change all the versions from 2.6 (when we used Docusaurus) to now to have a consistent system. Ideally all the URLs should be the same too.

One of the biggest problems with how we handle versions right now is the SEO is not the easiest thing to handle with duplicate content between different sites.

Docusaurus have a system built in for handling versioning, I am not sure how it works.
https://docusaurus.io/docs/versioning

This is a general conversation of how this should be handled because we are very open to other solutions.

**Comment by ricotrevisan** [comment]
A while back I was trying to integrate with some system and their documentation was versioned. I can't tell you how many WTFs I blurted out when -- after hours of work -- I realized I was relying on the outdated documentation.

I'm not sure we need to expose the version the published version of the docs. If someone really, really wants it, they could look in the repo?
